doc/Journal.tex

```diff
@@ -375,162 +345,214 @@ \subsection{Eigenvector Approach}
-These high values could result in the misidentification of the initial source position. 
-In contrast, one may see the lower off-diagonal values resulting from the binary and eigenvector approaches in Figs.~\ref{fig:EMAC} to \ref{fig:HadMAC}.
+It is worth pointing out a couple of key features for these MAC plots. 
+First, the diagonal entries correspond to comparing a vector to itself, and thus will always be equal to one.
+Second, the plots are symmetric about the diagonal.  
+Finally, the large off-diagonal regions are of particular interest because these high values could result in the mis-identification of the initial source direction. 
+The new proposed designs significantly lower off-diagonal values resulting in less degeneracy in the source direction as shown in Figs.~\ref{fig:EMAC} to \ref{fig:HadMAC}.
 
 \begin{figure}[ht!]
 \includegraphics[width={3.0in}]{../figs/EVMAC.pdf}
 \centering
-\caption{Representation of the eigenvector RSM MAC numbers for the basis $\theta=0$}
+\caption{Eigenvector RSM MAC values for the basis $\theta=0$.}
 \label{fig:EMAC}
 \end{figure}
 
 \begin{figure}[ht!]
 \includegraphics[width={3.0in}]{../figs/BiMAC.pdf}
 \centering
-\caption{Representation of the binary RSM MAC numbers for the basis $\theta=0$}
+\caption{Binary RSM MAC values for the basis $\theta=0$.}
 \label{fig:BMAC}
 \end{figure}
 
 \begin{figure}[ht!]
 \includegraphics[width={3.0in}]{../figs/HadMAC.pdf}
 \centering
-\caption{Representation of the Hadamard MAC numbers for the basis $\theta=0$}
+\caption{Hadamard MAC numbers for the basis $\theta=0$.}
 \label{fig:HadMAC}
 \end{figure}
 
-Note that the binary design may be improved by replacing the last four vectors with alternatives.  
-The adjacent off-diagonal terms in Figs.~\ref{fig:RSMMAC}, \ref{fig:BMAC}, and ~\ref{fig:HadMAC} show that there is a limit to the spacial resolution.  
-Specifically, for vector $p$, vectors $p+1$, $p+2$, and $p+3$ have MAC numbers that incrementally decrease.  
-This behavior indicates that information is shared by neighboring initial positions.  
+The adjacent off-diagonal terms in Figs.~\ref{fig:RSMMAC}, \ref{fig:BMAC}, and ~\ref{fig:HadMAC} show that there is a limit to the spatial resolution.  
+Specifically, vectors $p$, $p+1$, $p+2$, and $p+3$ have MAC numbers that incrementally decrease indicating that information is shared by neighboring initial positions.  
 In the presence of measurement noise, it may only be possible to identity the position to an accuracy that is a multiple of the $\theta$ or $\phi$ discretization.
 
+The off diagonal terms for the binary design are indicative of the limits of the design approach pursued.  
+Since a full optimization was not performed, it is likely that the binary design can be improved by replacing the last four vectors with alternatives. 
+This could result in a more rapid decrease close to the diagonal thereby improving the performance characteristics of this design class.
```

Comment:
It probably won't result in a more rapid decrease near the diagonal, but it will decrease the off-diagonal terms corresponding to these four vectors.